### DISCUSSION TITLE
mspider/intro/comp-archi-ui/namespaces/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3361] [Announcements]
# mspider/intro/comp-archi-ui/namespaces/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/intro/comp-archi-ui/namespaces/

**Comment by hanxiaop** [comment]
Istio 其实还有 `local-path-storage` 不会注入

**Reply by hanxiaop** [reply]
已经有了没看到。。

**Comment by hanxiaop** [comment]
页面好像错乱了，我在系统命名空间的留言跑到了这里，我在这里的留言跑到了 https://docs.daocloud.io/mspider/intro/app-spec/
